content/200-concepts/100-components/02-prisma-client/053-client-extensions/index.mdx

```diff
@@ -43,6 +43,8 @@ Extended clients interact with each other, and with the standard client, as foll
 - All extended clients and the standard client communicate with the same [Prisma query engine](/concepts/components/prisma-engines/query-engine).
 - All extended clients and the standard client share the same connection pool.
 
+> **Note**: The author of an extension can modify this behaviour since they're able to run arbitrary code as part of an extension. For example, an extension might actually create an entirely new `PrismaClient` instance (including its own query engine and connection pool). Be sure to check the documentation of the extension you're using to learn about any specific behaviour it might implement.
```

Comment:
```suggestion
> **Note**: The author of an extension can modify this behavior since they're able to run arbitrary code as part of an extension. For example, an extension might actually create an entirely new `PrismaClient` instance (including its own query engine and connection pool). Be sure to check the documentation of the extension you're using to learn about any specific behavior it might implement.
```